workflow/glsp-client/workflow-glsp/src/test-ui-extension/di.config.ts

```diff
@@ -0,0 +1,24 @@
+/********************************************************************************
```

Comment:
I guess this module was committed by accident?

Reply:
Right. Is removed now